sdk/core/azure-core/inc/azure/core/cryptography/hash.hpp

```diff
@@ -97,12 +97,12 @@ namespace Azure { namespace Core { namespace Cryptography {
     std::vector<uint8_t> Final() { return Final(nullptr, 0); };
```

Comment:
We included `<cstdint>`, which is guaranteed to declare `std::uint8_t` and _may_possibly_ declare `::uint8_t`, as opposed to `<stdint.h>` which _may_possibly_ declare `std::uint8_t` and is guaranteed to declare `::uint8_t`. To be consistent, we should either `std::`-qualify every `uint8_t` in this file, or change the include from `<cstdint>` to `<stdint.h>`.

Reply:
FWIW, we use `<cstdint>` everywhere else in the SDK without using `std::uint8_t`. Do you recommend we change all such instances? Is this something we should put in guidelines?

For example: 
https://github.com/Azure/azure-sdk-for-cpp/blob/2d4d766c6d454bfbd2a3627c5a3d8a4d81196480/sdk/core/azure-core/inc/azure/core/http/body_stream.hpp#L28
https://github.com/Azure/azure-sdk-for-cpp/blob/2d4d766c6d454bfbd2a3627c5a3d8a4d81196480/sdk/core/azure-core/inc/azure/core/http/body_stream.hpp#L87

https://github.com/Azure/azure-sdk-for-cpp/blob/2d4d766c6d454bfbd2a3627c5a3d8a4d81196480/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp#L7
https://github.com/Azure/azure-sdk-for-cpp/blob/2d4d766c6d454bfbd2a3627c5a3d8a4d81196480/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp#L114